### DISCUSSION TITLE
POST APIs Simplest Method in Flutter

### DISCUSSION THREAD

**Discussion by Ali-Murtaza143** [community/community#137181] [Programming Help]
### Select Topic Area

Show & Tell

### Body

I want to know POST APIs simplest method in Flutter.

**Comment by ShahzainAhmed** [comment]
This is how you can write POST APIs in simplest possible way!

static postApi() async {
    final uri = Uri.parse("https://reqres.in/api/register");
    final response = await http.post(
      uri,
      body: {
        "email": "eve.holt@reqres.in",
        "password": "pistol",
      },
    );

    if (response.statusCode == 200) {
      print('Response postApi(): ${response.body}');
    } else {
      print('Failed to load data. Status code: ${response.statusCode}');
    }
  }

**Comment by ebndev** [comment]
Please see [this comment](https://github.com/orgs/community/discussions/137180#discussioncomment-10607513) for reference to why this discussion was closed and locked. Thanks!
